### ISSUE TITLE
Upgrade Angular

### ISSUE THREAD

**Issue by Faheetah** [allen13/golerta#6]
@allen13  We're using an old version of Angular 1 and it's pretty behind on best practices too. Want me to look into upgrading to Angular 2 or maybe even porting to React? I've been digging into both of those lately and shouldn't be terribly hard to port over.

**Comment by allen13** [comment]
Absolutely! It would be much appreciated! I’m framework agnostic as long as it is better than the current implementation.
